packages/@adobe/react-spectrum/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Spectrum UI components in React",
   "license": "Apache-2.0",
   "main": "dist/main.js",
+  "exports": "./dist/main.js",
```

Comment:
This means any tool using the exports field will use the commonjs version. Maybe we need import and require sub-fields?

Reply:
You mean like this right?
```
  "exports": {
    "import": "./dist/main.js",
    "require": "./dist/module.js"
  },
```
or is there something else I need to do?

Reply:
What's a good test app that would use this?